apps/src/templates/policy_compliance/NewStudentAccountConsent.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import * as color from '../../util/color';
+
+function permissionGrantedMessage(date) {
+  const grantedDateString = i18n.newStudentAccountConsentValidPermissionGranted(
+    {
+      date: date.toLocaleString(),
```

Comment:
Tiny nitpick, but there's also a toLocaleDateString() that should omit the time. Imo it looks slightly off to say "Permission granted on 6/7/23, 9:42:13" when the exact time down to the second isn't really relevant.

Reply:
I do this in my lockout panel!

Without time: https://github.com/code-dot-org/code-dot-org/blob/30b1392e97c7e423c15808fc38d772346bc36a19/apps/src/templates/sessions/LockoutPanel.jsx#L90

With time: https://github.com/code-dot-org/code-dot-org/blob/30b1392e97c7e423c15808fc38d772346bc36a19/apps/src/templates/sessions/LockoutPanel.jsx#L155

Reply:
I do the timezone to UTC since it needs to be the day and if it happens to be close to the date line it might report the wrong date in certain locales... we just have to ensure we set the datetime for delete or calculate it so it works in UTC. not sure if that is relevant here.

Reply:
@wilkie Aren't we good here since the user would probably want the date in their own local timezone? Even though it might be the "wrong" date in UTC, it's the right date in the user's local time. Agree that for the delete date we need to make sure to do our date math correctly though.

Reply:
Yes you don't here, I don't think.

Reply:
Fixed, thanks for the tip. It looks a lot better.